utils/dbutils.py

```diff
@@ -0,0 +1,151 @@
+from gempython.utils.gemlogger import getGEMLogger, printYellow
+
+import pandas as pd
+
+dbNameDev = "INT2R" # development
+dbNamePro = "cms_omds_lb" # production
+
+connection = "oracle://CMS_GEM_APPUSER_R:GEM_Reader_2015@"
```

Comment:
Standard comment about hard-coding credentials in committed files...

Reply:
I thought that since it was readonly it wouldn't be an issue.

What would your ideal solution look like?  An `os.getenv("DB_VAR")` or something similar that is defined on the host machine?

Reply:
Either an environment variable, or a configuration file can be stored either user-local, or system (or both, and a preference is made for the user local one)
Env var is probably simplest at present